### PR TITLE
Add edit role to all revelant roles

### PR DESCRIPTION
**Pull request by cibelius**
## Description

Core PR: https://github.com/opencrvs/opencrvs-core/pull/11275

Also add translations.

## Checklist

- [x] I have linked the correct Github issue under "Development"
- [x] I have tested the changes locally, and written appropriate tests
- [x] I have tested beyond the happy path (e.g. edge cases, failure paths)
- [ ] I have updated the changelog with this change (if applicable)
- [x] I have updated the GitHub issue status accordingly

### CODE REVIEW

File: src/translations/client.csv [hunk 1]
```diff
@@ -800,6 +801,12 @@ event.action.declare.form.section.person.field.age.error,Error message for inval
...
+event.edit.registerWithEdits.label,Label for "Register with edits" in edit action menu,Register with edits,Enregistrer avec modifications
+event.edit.declareWithEdits.label,Label for "Declare with edits" in edit action menu,Declare with edits,Déclarer avec modifications
+event.edit.registerWithEdits.description,Description for "Register with edits" in edit action menu,Are you sure you want to register this event with these edits?,Êtes-vous sûr de vouloir enregistrer cet événement avec ces modifications ?
+event.edit.declareWithEdits.description,Description for "Declare with edits" in edit action menu,Are you sure you want to edit this declaration? By confirming you are redeclaring this event and override past changes...,Êtes-vous sûr de vouloir modifier cette déclaration ? En confirmant, vous redéclarez cet événement et remplacez les modifications précédentes...
+event.edit.comment.label,The label for the comment,Comments,Commentaires
+editPageBanner.message,The message for the edit page banner,You are editing a record declared by {name} ({role} at {location}),Vous modifiez un dossier déclaré par {name} ({role} à {location})
```
Comment (Nil20) [inline src/translations/client.csv]: maybe run the pre-commit correctly before merging, to have the translations sorted

**Review by Nil20 — APPROVED** [review]
